test/jdk/sun/security/tools/jarsigner/compatibility/Compatibility.java

```diff
@@ -1034,13 +1035,19 @@ private static OutputAnalyzer execTool(String toolPath, String... args)
             throws Throwable {
         long start = System.currentTimeMillis();
         try {
+            String[] cmd;
+            if (Platform.isWindows()) {
+                cmd = new String[args.length + 3];
+                System.arraycopy(args, 0, cmd, 3, args.length);
+            } else {
+                cmd = new String[args.length + 4];
+                cmd[3] = "-J-Djava.security.egd=file:/dev/./urandom";
```

Comment:
IIRC, this is a workaround for an old issue which no longer applies after JDK-6425477.  By default NativePRNG generates from /dev/urandom, only grabbing seed data from /dev/random.

I believe the egd workaround can be completely removed unless there's something that asks for NativePRNGBlocking.

Reply:
SHA1PRNG, which is no longer the default on Linux.  But some apps are still asking for SHA1PRNG instead of using the default NativePRNG on Linux.

Reply:
FWIW, NativePRNGBlocking ignores java.security.egd and always uses `/dev/random`.

I'd rather not clean up all references to EGD here. Filed [JDK-8293412](https://bugs.openjdk.org/browse/JDK-8293412) as a follow up.

Reply:
So IIUC, for this PR you're only going to tweak so that egd is not set in Windows, and then JDK-8293412 will address this comment and other instances throughout the base?

Reply:
Yes, that's the plan.